completions/git.completion.sh

```diff
@@ -10,15 +10,11 @@ _omb_completion_git_initialize() {
             do
                 local _file="$prefix/$file"
                 if [[ -f $_file && -r $_file ]]; then
-                    is_completion_loaded=1
                     source "$_file"
-                    break 2
+                    return
                 fi
             done
         done
-        if [[ $is_completion_loaded == 0 ]]; then
-            source "$OSH/tools/git.completion.bash"
-        fi
```

Comment:
I think you forgot to leave the line `source "OSH/tools/git.completion.bash"`. I'll include it at my side.